patches/@lerna+conventional-commits+3.18.5.dev.patch

```diff
@@ -1,22 +0,0 @@
-diff --git a/node_modules/@lerna/conventional-commits/lib/recommend-version.js b/node_modules/@lerna/conventional-commits/lib/recommend-version.js
-index f13ae56..ae5a449 100644
---- a/node_modules/@lerna/conventional-commits/lib/recommend-version.js
-+++ b/node_modules/@lerna/conventional-commits/lib/recommend-version.js
-@@ -51,7 +51,16 @@ function recommendVersion(pkg, type, { changelogPreset, rootPath, tagPrefix, pre
- 
-         // result might be undefined because some presets are not consistent with angular
-         // we still need to bump _something_ because lerna saw a change here
--        const releaseType = data.releaseType || "patch";
-+        let releaseType = data.releaseType || "patch";
-+        
-+        // Don't gratuitously break compatibility with clients using `^0.x.y`.
```

Comment:
Is this logic in `lerna@4.0.0`?  We needed it so that our pre-`1.x.x` packages didn't get bumped up to `1.x.x` just because they had a breaking change.

Reply:
Yes, I mentioned this in [e78169b](https://github.com/Agoric/agoric-sdk/pull/4646/commits/e78169b7b8206973b9a15924996f4d1ac312bb0e) but should have mentioned in PR too:
> it includes what we were patching: https://github.com/lerna/lerna/pull/2486

Reply:
Vindication!